### PR TITLE
update what is tizen

### PR DESCRIPTION
**Pull request by jhchoilee**
### Change Description ###

Update tizen document
- update overview & profile document
- update image files (add new, delete old image)
- etc

### CODE REVIEW

File: docs/platform/what-is-tizen/overview.md [hunk 1]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
-- Tablets: DIGMA Plane 8501 3G
+- TV & Audio : Samsung Smart TVs, Samsung Lyfestyle TV, Samsung Smart Monitor, Samsung Sound Bar
+- Display : Samsung Smart Signage, Samsung Flip, Samsung Kiosk
+- Smart appliances: Samsung Family Hub Fridge, Samsung air conditioner, Samsung washing machine
 - Wearables: Samsung Galaxy Watch series (Galaxy Watch, Galaxy Watch Active, Galaxy Watch Active2, Galaxy Watch3), Gear series (Gear 2, Gear 2 Neo, Gear S, Gear S2, Gear S3)
-- Smart TVs: Samsung SUHD, UHD, LED TVs
-- Cameras: Samsung NX-series, Gear 360
-- Smart appliances: Samsung Family Hub Fridge
+- Mobile phones: Samsung Z series (Z1(3G), Z3(3G/LTE), Z2(LTE), Z4)
+- Tablets: DIGMA Plane 8501 3G
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:52]:
```suggestion
- Tablets: Digma Plane 8501 3G
```

File: docs/platform/what-is-tizen/overview.md [hunk 2]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
+- Cameras: Samsung NX-series(NX300, NX30, NX2000, NX1, NX500), Gear 360
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:53]:
```suggestion
- Cameras: Samsung NX series (NX300, NX30, NX2000, NX1, NX500), Gear 360
```

File: docs/platform/what-is-tizen/overview.md [hunk 3]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
+- OBD : Samsung Connect Auto
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:54]:
```suggestion
- OBD: Samsung Connect Auto
```

File: docs/platform/what-is-tizen/overview.md [hunk 4]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
+
 
-As of Oct. 2021, there are about 200 million Tizen devices in the world. In addition, Samsung is holding the highest market share in the global TV market and its recent models are all based on the Tizen platform. It is expected for the Tizen platform to broaden its ecosystem with increasing market requirements and user demands.
+Samsung is holding the highest market share in the global TV market and its recent models are all based on the Tizen platform. It is expected for the Tizen platform to broaden its ecosystem with increasing market requirements and user demands.
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/overview.md:57]:
```suggestion
Samsung holds the highest market share in the global TV market and its recent models are all based on the Tizen platform. It is expected that the Tizen platform to broaden its ecosystem with increasing market requirements and user demands.
```

File: docs/platform/what-is-tizen/profiles/tv.md [hunk 5]
```diff
@@ -27,17 +27,12 @@ Samsung has completely redesigned its Smart TV experience. With a new intuitive
...
 The Samsung Smart TV SDK combines TV features with web-based technologies. The SDK provides tools for developing Samsung TV applications for the Tizen platform.
 
-To develop applications for the TV profile, refer to the following:
+To develop applications for Tizen TV, refer to the following:
 
 - [Smart TV Quick-start Guide](http://developer.samsung.com/tv/develop/getting-started/quick-start-guide){:target="_blank"}
-- [**Create Samsung Smart TV Web applications**](../../../application/web/get-started/tv/first-samsung-tv-app.md)
-- [Create TV .NET Applications](../../../application/dotnet/get-started/first-app.md)
-- [Create TV Web applications](../../../application/web/get-started/tv/first-app.md)
+- [Create Tizen TV .NET Applications](https://developer.samsung.com/smarttv/develop/getting-started/creating-tv-applications.html)
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/profiles/tv.md:33]:
```suggestion
- [Create Tizen TV .NET Applications](https://developer.samsung.com/smarttv/develop/getting-started/creating-tv-applications.html){:target="_blank"}
```

File: docs/platform/what-is-tizen/profiles/tv.md [hunk 6]
```diff
@@ -27,17 +27,12 @@ Samsung has completely redesigned its Smart TV experience. With a new intuitive
...
+- [Create Tizen TV TV Web applications](https://developer.samsung.com/smarttv/develop/tizen-net-tv/getting-started/creating-net-tv-applications.html)
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/profiles/tv.md:34]:
```suggestion
- [Create Tizen TV TV Web applications](https://developer.samsung.com/smarttv/develop/tizen-net-tv/getting-started/creating-net-tv-applications.html){:target="_blank"}
```

File: docs/platform/what-is-tizen/profiles/tv.md [hunk 7]
```diff
@@ -47,4 +42,4 @@ For more information on developing a Samsung Smart TV application, go to [SAMSUN
 
 **Figure: Samsung Smart TVs**
 
-![Samsung Smart TVs](media/profile_tv_devices.jpg)
+![Samsung Smart TVs](media/profile_tv_devices1.jpg)
```
Comment (omar-srbd) [inline docs/platform/what-is-tizen/profiles/tv.md:45]:
```suggestion
![Samsung Smart TVs](media/profile_tv_devices1.jpg){:target="_blank"}
```

**Review by omar-srbd — COMMENTED** [review]
Hello @jhchoilee, I have reviewed the files in this PR and left some correction suggestions.

Also, please update the _docs/application/toc_all.md_ file to include the new file _docs/platform/what-is-tizen/profiles/tizen_custom.md_ so that it will be added to the LNB.

Thank you.

File: docs/platform/what-is-tizen/overview.md [hunk 8]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
+As diverse devices are getting smarter and more connected, there is a strong need for a software platform for integrated services with connected products. Tizen is designed for this and has been evolving to connect everything.
+
+To support different types of devices with a seamless user experience, Tizen provides the features and functions for various kind of devices. Tizen supports both headed and headless products. In the near future, lots of internet connected devices are anticipated to be on the market and IoT services are expected to pervade everyone's daily life. Tizen is moving forward to the future of everything connected.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:10]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 9]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
-For AI services, Tizen provides a framework for offloading AI inference computation to connected high-end devices. Even when a device doesn't have enough computing power to run AI models, a service can find an alternative device to run the model among connected devices. With this feature, every Tizen device will be able to run AI services by leveraging the capability of peer devices.
+Tizen is a Linux-based, open source software platform. Anyone can contribute and freely develop the system software and applications, which is beneficial for both device manufacturers and application developers. Furthermore, by supporting web applications with open web technologies and C# applications with .Net technologies, Tizen provides developers flexibility and extensibility. Based on the open ecosystem, the user can expect various experiences from Tizen devices and services. With its openness, Tizen can pursue continuing progress and expand the ecosystem.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:18]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 10]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
 
+Tizen is built to work on diverse devices.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:20]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 11]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
-- Run various types of models with Machine Learning API.
-- Use H/W acceleration with NN Runtime.
+.
 ## Convergence platform for the emerging era
 
 Since 2012, Tizen has been expanding as smart gadgets have become generalized and connected. By accommodating various device types, Tizen is gearing up for the new era of convergence. As of now, there is a wide variety of Tizen devices on the market:
 
-- Mobile phones: Samsung Z series (Z1, Z2, Z3, Z4)
-- Tablets: DIGMA Plane 8501 3G
+- TV & Audio : Samsung Smart TVs, Samsung Lyfestyle TV, Samsung Smart Monitor, Samsung Sound Bar
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:47]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 12]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
 
-- Provides an optimized execution combination of various open kernels and private kernels based on a proprietary algorithm developed in-house.
-   - Kernels from open sources such as ARM Compute Library (ACL), Ruy, and XNNPACK, and customized improvements according to needs. 
-- Support for dynamic tensors whose shape keeps changing during inference.
-- Support for models with control flow operators. (IF, WHILE)
-- Provides various executors and is expandable.
-   - Linear executor.
-   - Parallel executor using CPU and GPU together.
-- Partitioning and multithreading of neural network models at runtime to improve overall inference throughput.
+Tizen custom device can be created by combining API building blocks in various ways as per the requirements. Required packages are installed based on the dependencies related to the specified building blocks.
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:32]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 13]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
-- Use H/W acceleration with NN Runtime.
+.
 ## Convergence platform for the emerging era
 
 Since 2012, Tizen has been expanding as smart gadgets have become generalized and connected. By accommodating various device types, Tizen is gearing up for the new era of convergence. As of now, there is a wide variety of Tizen devices on the market:
 
-- Mobile phones: Samsung Z series (Z1, Z2, Z3, Z4)
-- Tablets: DIGMA Plane 8501 3G
+- TV & Audio : Samsung Smart TVs, Samsung Lyfestyle TV, Samsung Smart Monitor, Samsung Sound Bar
+- Display : Samsung Smart Signage, Samsung Flip, Samsung Kiosk
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:48]: applied

File: docs/platform/what-is-tizen/overview.md [hunk 14]
```diff
@@ -5,141 +5,69 @@ Tizen is an open source operating system based on Linux, hosted by the Linux Fou
...
+- Smart appliances: Samsung Family Hub Fridge, Samsung air conditioner, Samsung washing machine
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:49]: applied

File: docs/platform/what-is-tizen/overview.md (hunk 1 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:52]: applied

File: docs/platform/what-is-tizen/overview.md (hunk 2 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:53]: applied

File: docs/platform/what-is-tizen/overview.md (hunk 3 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:54]: applied

File: docs/platform/what-is-tizen/overview.md (hunk 4 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/overview.md:57]: applied

File: docs/platform/what-is-tizen/profiles/tv.md (hunk 5 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/profiles/tv.md:33]: applied

File: docs/platform/what-is-tizen/profiles/tv.md (hunk 6 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/profiles/tv.md:34]: applied

File: docs/platform/what-is-tizen/profiles/tv.md (hunk 7 above)
Comment (jhchoilee) [reply docs/platform/what-is-tizen/profiles/tv.md:45]: applied

**Comment by jhchoilee** [comment]
Thanks for review.
I updated document based on the review comment.
including "please update the docs/application/toc_all.md file to include the new file docs/platform/what-is-tizen/profiles/tizen_custom.md"

**Review by omar-srbd — COMMENTED** [review]
@jhchoilee, please check the suggestion.

Thank you.

File: docs/platform/what-is-tizen/profiles/tv.md [hunk 15]
```diff
@@ -47,4 +42,4 @@ For more information on developing a Samsung Smart TV application, go to [SAMSUN
 
 **Figure: Samsung Smart TVs**
 
-![Samsung Smart TVs](media/profile_tv_devices.jpg)
+![Samsung Smart TVs](media/profile_tv_devices1.jpg){:target="_blank"}
```
[the remark this answers is not in the document]
Comment (jhchoilee) [reply docs/platform/what-is-tizen/profiles/tv.md]: applied

**Comment by jhchoilee** [comment]
I updated as follows:
Samsung Smart TVs{:target="_blank"}
--> Samsung Smart TVs

Thanks

**Review by omar-srbd — APPROVED** [review]
@jhchoilee, thanks for incorporating all the changes.

@asm-foysal, I am approving this PR for merging.

**Pull request merged**